packages/plasma-new-hope/src/examples/plasma_b2c/components/Accordion/Accordion.stories.tsx

```diff
@@ -91,3 +93,80 @@ export const Default: StoryObj<AccordionProps> = {
         );
     },
 };
+
+const getSizeForIcon = (size) => {
+    const map = {
```

Comment:
можно за функцию вынести

Reply:
хотя, это сторибук...

Reply:
Тут не понял момент с нестандартными размерами типа `xsr`, `sr`, `mr` и `lr`, поскольку их нет у аккордеона. 

Кажется, что здесь достаточно вот такого:
```suggestion
const getSizeForIcon = (size) => size === 'xs' ? 'xs' : 's'
```

Reply:
сорри, но я так и не понял, зачем мы в мапе используем ключи `xsr`, `sr`, `mr `и `lr`, если у аккордеона размеры только `xs`, `s`, `m` и `l`

Reply:
согласен, ща